### PR TITLE
Add `Polymer.dom.deepContains`

### PR DESCRIPTION
**Pull request by azakus**
- Also add tests

### CODE REVIEW

**Comment by azakus** [comment]
Related to PolymerElements/paper-behaviors#43

**Comment by ebidel** [comment]
Pingy ping. This is currently breaking our ripples on the Google Codelab site.

File: test/unit/polymer-dom.js [hunk 1]
```diff
@@ -917,4 +917,21 @@ suite('Polymer.dom non-distributed elements', function() {
     assert.equal(Polymer.dom(document.createElement('div')).getDestinationInsertionPoints().length, 0);
     assert.equal(Polymer.dom(document).getDestinationInsertionPoints().length, 0);
   });
+
+  test('Deep Contains', function() {
```
Comment (sorvell) [inline test/unit/polymer-dom.js]: Let's add a couple tests for non-distributed children since we have code that covers this case.

**Comment by sorvell** [comment]
LGTM :+1:

**Pull request merged**